src/components/binding/binding.js

```diff
@@ -150,13 +162,16 @@ genotet.BindingView = function(viewName, params) {
     .on('genotet.gene', function(event, data) {
       this.data.tracks[data.trackIndex].fileName = data.fileName;
       this.loader.loadFullTrack(data.trackIndex, data.fileName,
-        this.data.chr);
+        this.data.chr, false);
     }.bind(this));
 
   $(this.loader)
     .on('genotet.chr', function(event, chr) {
       this.panel.updateChr(chr);
     }.bind(this))
+    .on('genotet.addTrack', function() {
+      this.loader.loadBindingList();
```

Comment:
1. If the binding files change after uploading files, then binding list should be requested after file uploads.

2. loadBindingList() simply fetches the list of binding data. I still don't see why it is related with addTrack.

3. It seems to me that updateTracks (maybe updatePanelTracks is clearer) simply updates the panel which should not be sending further signal at all. I don't see why there is an event loop.
addTrack -> updatePanelTracks
loadBindingList -> updatePanelTracks